presets/minimal/phase0.yaml

```diff
@@ -86,3 +86,8 @@ MAX_ATTESTATIONS: 128
 MAX_DEPOSITS: 16
 # 2**4 (= 16)
 MAX_VOLUNTARY_EXITS: 16
+
+# Networking
+# ---------------------------------------------------------------
+# 2**8 (= 256)
+EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
```

Comment:
I guess that if this one is too low, the churn rate will be too high. So, no network probably wants this number to be too low.

That is, around 1/EPOCHS_PER_SUBNET_SUBSCRIPTION of the nodes will change the subnets in each epoch.

However, I just guess.

Reply:
Because our previous `EPOCHS_PER_*` settings are in presets. e.g, `EPOCHS_PER_ETH1_VOTING_PERIOD`, `EPOCHS_PER_SLASHINGS_VECTOR`, `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`. It seems less likely we need to tune it for testing the networking configurations.

I can move it to configs if you prefer.

Reply:
ah but the other `EPOCHS_PER_` are things that impact the shape of merkle trees/state transitions.

This seems a bit more like something you could mess with on a network/testnet without affecting compile time build in a negative way